README.md

```diff
@@ -153,19 +154,29 @@ coffee_average_per_person = []
 
 ```
 
+             
 ### Answer
+Ahlam = [7.43,14.65,4,43]
+ahlam.sum.fdiv(arr.size)
+Sulaimam = [800,14.65,4,43]
+sulaiman.sum.fdiv(arr.size)
+Norah= [7.43,100,4.43]
+norah.sum.fdiv(arr.size)
+end
```

Comment:
The main idea behind the question was to try practicing how to access hashes and arrays within each other. Creating an array with the prices defies the purpose.